test/e2e/gardenadm/mediumtouch/exec.go

```diff
@@ -71,3 +78,36 @@ func Run(args ...string) *gexec.Session {
 func RunAndWait(ctx context.Context, args ...string) *gexec.Session {
 	return Wait(ctx, Run(args...))
 }
+
+// RunInMachine runs gardenadm in the given machine (sorted lexicographically) with the given arguments and returns the
+// gbytes.Buffers.
+func RunInMachine(ctx context.Context, technicalID string, ordinal int, args ...string) (*gbytes.Buffer, *gbytes.Buffer, error) {
+	var stdOutBuffer, stdErrBuffer = gbytes.NewBuffer(), gbytes.NewBuffer()
+
+	err := RuntimeClient.PodExecutor().ExecuteWithStreams(
+		ctx,
+		technicalID,
+		machinePodName(ctx, technicalID, ordinal),
+		ContainerName,
+		nil,
+		io.MultiWriter(stdOutBuffer, gexec.NewPrefixedWriter("[machine][out] ", GinkgoWriter)),
```

Comment:
Prefixing the output with the machine name probably simplifies debugging test failures.